### PR TITLE
feat: or-852 terminate vlimperskey on termination of organisation

### PR DESCRIPTION
**Pull request by W0dan**

### CODE REVIEW

File: src/OrganisationRegistry/Organisation/OrganisationCommandHandlers.cs [hunk 1]
```diff
@@ -933,6 +933,7 @@ private void ThrowIfCircularRelationshipDetected(Organisation organisation, Peri
                         _organisationRegistryConfiguration.Kbo.OrganisationCapacityTypeIdsToTerminateEndOfNextYear,
                         _organisationRegistryConfiguration.Kbo.OrganisationClassificationTypeIdsToTerminateEndOfNextYear,
                         _organisationRegistryConfiguration.Kbo.FormalFrameworkIdsToTerminateEndOfNextYear,
+                        _organisationRegistryConfiguration.VlimpersKeyTypeId,
```
Comment (koenmetsu) [inline src/OrganisationRegistry/Organisation/OrganisationCommandHandlers.cs:936]: This deviates from previous code. let's discuss thistomorrow.

File: test/OrganisationRegistry.UnitTests/OrganisationRegistry.UnitTests.csproj [hunk 2]
```diff
@@ -21,6 +21,6 @@
   <ItemGroup>
     <Folder Include="MagdaResponses" />
   </ItemGroup>
-
+
```
Comment (koenmetsu) [inline test/OrganisationRegistry.UnitTests/OrganisationRegistry.UnitTests.csproj:24]: whiteline

**Review by koenmetsu — COMMENTED** [review]
Let's go over the rest tomorrow. :+1:

File: src/OrganisationRegistry/Organisation/OrganisationCommandHandlers.cs (hunk 1 above)
Comment (W0dan) [reply src/OrganisationRegistry/Organisation/OrganisationCommandHandlers.cs]: we just need to cleanup the config side of things ...

**Pull request merged**